torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -1259,6 +1260,19 @@ Error PyTorchModelLoader::loadRelu(const torch::jit::Node *ptNode) {
   return addValueMapping(outputs[0], rescaleIntToUint(glowNode->getResult()));
 }
 
+Error PyTorchModelLoader::loadTanh(const torch::jit::Node *ptNode) {
+  auto inputs = ptNode->inputs();
+  auto outputs = ptNode->outputs();
+  RETURN_IF_ERR(checkInputAndOutputSizes(inputs, 1, outputs, 1));
+
+  glow::NodeValue input;
+  ASSIGN_VALUE_OR_RETURN_ERR(input, getGlowNodeValueForValue(inputs[0]));
+  input = rescaleUIntToInt(input);
```

Comment:
Why do we do this?

Reply:
Oh it's for quantized inputs, I copied it accidentally grabbed it from relu. I'll remove it and if someone needs it for quantized op they can add it later